site/cypress/e2e/Cypress-UI/navigation_page.spec.js

```diff
@@ -9,6 +7,91 @@ function validate_navigation_page_sections(sections) {
     });
 }
 
+const white = 'rgb(255, 255, 255)';
+const blue = 'rgb(0, 99, 152)';
+const red = 'rgb(220, 53, 69)';
+const green = 'rgb(95, 183, 96)';
+const buttonOrder = ['team-btn', 'submit-btn', 'grade-btn', 'quick-link-btn'];
+
+function checkButtons(courseID, buttonText, buttonColor) {
```

Comment:
I think that `course` should be `gradeable` for this PR, as `course` in this case would be `Sample`.